src/category_theory/groupoid/subgroupoid.lean

```diff
@@ -218,10 +263,30 @@ lemma mem_discrete_iff {c d : C} (f : c ⟶ d):
   (f ∈ (discrete).arrows c d) ↔ (∃ (h : c = d), f = eq_to_hom h) :=
 ⟨by { rintro ⟨⟩, exact ⟨rfl, rfl⟩ }, by { rintro ⟨rfl, rfl⟩, split }⟩
 
-/-- A subgroupoid is normal if it is “wide” (meaning that its carrier set is all of `C`)
-    and satisfies the expected stability under conjugacy. -/
-structure is_normal : Prop :=
+/-- A subgroupoid is wide if its carrier set is all of `C`-/
+structure is_wide : Prop :=
 (wide : ∀ c, (𝟙 c) ∈ (S.arrows c c))
+
+lemma is_wide_iff_objs_eq_univ : S.is_wide ↔ S.objs = ⊤ :=
+begin
+  split,
+  { rintro h,
+    ext, split; simp only [top_eq_univ, mem_univ, implies_true_iff, forall_true_left],
+    apply mem_objs_of_src S (h.wide x), },
+  { rintro h,
+    refine ⟨λ c, _⟩,
+    obtain ⟨γ,γS⟩ := (le_of_eq h.symm : ⊤ ⊆ S.objs) (set.mem_univ c),
+    exact id_mem_of_src S γS, },
+end
+
+lemma id_mem_of_is_wide (Sw : S.is_wide) (c : C) : (𝟙 c) ∈ S.arrows c c := Sw.wide c
```

Comment:
I've renamed but kept the lemma and the original `is_wide.wide` simply because (in my perspective `1 c \in S.arrows c c` is kind of an "internal encoding" for being wide, if that makes sense.